Automattic/sensei
# REST API: Add Importer base endpoints

### Changes proposed in this Pull Request

* This adds the basic `GET`, `POST`, and `DELETE` endpoints for the importer.
* This doesn't handle the `PUT` request because right now the only job options will be the uploaded files, handled by a future sub-endpoint in the `Sensei_REST_API_Import_Controller` class.
* Schema is pretty bare right now, but it can include additional things as needed.

### Endpoints Added
- `GET /wp-json/sensei-internal/v1/import`
- `POST /wp-json/sensei-internal/v1/import`
- `DELETE /wp-json/sensei-internal/v1/import`

### Testing instructions

Using your favorite REST API client:
* `GET /wp-json/sensei-internal/v1/import` should return 404 if job not created.
* `POST /wp-json/sensei-internal/v1/import` should return 201 if first created and 200 if already created (same as `GET`). Should return standard job schema (just `id` and `status` for now).
* `GET /wp-json/sensei-internal/v1/import` should now return 200 and same job as `POST` above.
* `DELETE /wp-json/sensei-internal/v1/import` should cancel the job.
* `GET /wp-json/sensei-internal/v1/import` should once again 404.
